module/cyphersystem.js

```diff
@@ -575,18 +575,18 @@ function quickRollMacro(title) {
 
 function easedRollMacro() {
   let d = new Dialog({
-    title: "Eased Stat Roll",
-    content: `<div align="center"><label style='display: inline-block; text-align: right'><b>Eased by: </b></label>
+    title: game.i18n.localize("CYPHERSYSTEM.EasedStatRoll"),
+    content: `<div align="center"><label style='display: inline-block; text-align: right'><b>${game.i18n.localize("CYPHERSYSTEM.EasedBy")}: </b></label>
```

Comment:
This needs to be capitalized, but isn’t in the string. We might need this string twice, once capitalized, once not.

Reply:
That is why at the beginning I used the `toLowerCase` for `Eased by/Hindered by` so we do not need two different strings.

Reply:
Refactored translation